src/m365/spo/commands/list/list-sensitivitylabel-ensure.spec.ts

```diff
@@ -0,0 +1,255 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+import { Cli } from '../../../../cli/Cli';
+import { CommandInfo } from '../../../../cli/CommandInfo';
+import { Logger } from '../../../../cli/Logger';
+import Command, { CommandError } from '../../../../Command';
+import request from '../../../../request';
+import { pid } from '../../../../utils/pid';
+import { session } from '../../../../utils/session';
+import { sinonUtil } from '../../../../utils/sinonUtil';
+import commands from '../../commands';
+const command: Command = require('./list-sensitivitylabel-ensure');
+
+describe(commands.LIST_SENSITIVITYLABEL_ENSURE, () => {
+  const webUrl = 'https://contoso.sharepoint.com';
+  const name = 'Label';
+  const listTitle = 'Shared Documents';
+  const listId = 'b4cfa0d9-b3d7-49ae-a0f0-f14ffdd005f7';
+  const listUrl = '/Shared Documents';
+  const sensitivityLabelId = '0d39dc11-75ff-4309-8b32-ff94f0e41607';
+  const graphResponse = {
+    "value": [
+      {
+        "id": sensitivityLabelId,
+        "name": "Label",
+        "description": "",
+        "color": "",
+        "sensitivity": 7,
+        "tooltip": "sensitive information.",
+        "isActive": true,
+        "isAppliable": true
+      }
+    ]
+  };
+
+  let log: any[];
+  let logger: Logger;
+  let commandInfo: CommandInfo;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
+    sinon.stub(pid, 'getProcessName').callsFake(() => '');
+    sinon.stub(session, 'getId').callsFake(() => '');
+    auth.service.connected = true;
+    commandInfo = Cli.getCommandInfo(command);
+  });
+
+  beforeEach(() => {
+    log = [];
+    logger = {
+      log: (msg: string) => {
+        log.push(msg);
+      },
+      logRaw: (msg: string) => {
+        log.push(msg);
+      },
+      logToStderr: (msg: string) => {
+        log.push(msg);
+      }
+    };
+  });
+
+  afterEach(() => {
+    sinonUtil.restore([
+      request.get,
+      request.patch
+    ]);
+  });
+
+  after(() => {
+    sinonUtil.restore([
+      auth.restoreAuth,
+      appInsights.trackEvent,
+      pid.getProcessName,
+      session.getId
+    ]);
```

Comment:
Hi, just popping in here to say that we should refactor this `sinonUtil.restore([...` to `sinon.restore()`. This is a new way to ensure that the entire sandbox is cleared and will from now on be used in every command.